### DISCUSSION TITLE
Bug Encountered

### DISCUSSION THREAD

**Discussion by ghost** [DEMOLITIONDON96/Demolition-Engine#64] [Source Q&A] [answered]
@DEMOLITIONDON96 I got this while compiling.
`source/PlayState.hx:4912: characters 2-9 : Expected }`

**Comment by Theoyeah** [comment]
Oh oh  ooo

**Comment by ThatOneSillyGuyDon** [comment, marked as answer]
just fixed it rn, for I only had to add in a few missing brackets here and there, according for my build, in which it functions with the 2 new options as of now
